imap_processing/codice/codice_l1a.py

```diff
@@ -87,29 +89,23 @@ def __init__(self, table_id: int, plan_id: int, plan_step: int, view_id: int):
         self.plan_step = plan_step
         self.view_id = view_id
 
-    def create_science_dataset(self, packets: list) -> xr.Dataset:
+    def create_science_dataset(self, start_time: np.datetime64) -> xr.Dataset:
         """Create an ``xarray`` dataset for the unpacked science data.
 
         The dataset can then be written to a CDF file.
 
         Parameters
         ----------
-        packet : list[space_packet_parser.parser.Packet]
-            List of packets for the APID of interest
+        start_time : np.datetime64
+            The start time of the packet, used to determine epoch data variable
 
         Returns
         -------
         xr.Dataset
             ``xarray`` dataset containing the science data and supporting metadata
         """
         epoch = xr.DataArray(
-            [
-                calc_start_time(
-                    packet.data["ACQ_START_SECONDS"].raw_value,
-                    launch_time=np.datetime64("2010-01-01T00:01:06.184", "ns"),
-                )
-                for packet in packets
-            ],
+            [start_time],
```

Comment:
Yeah, I think this will likely change post SIT-3 when I work on getting the pipeline to read in multiple APIDs. But right now the pipeline handles one-APID-per-file, so there is only one start time.